tsl/test/expected/move.out

```diff
@@ -0,0 +1,469 @@
+        
+                               QUERY PLAN                                
+-------------------------------------------------------------------------
+ Append
+   ->  Bitmap Heap Scan on _hyper_1_2_chunk
+         Recheck Cond: ("time" = 7)
+         ->  Bitmap Index Scan on _hyper_1_2_chunk_cluster_test_time_idx
+               Index Cond: ("time" = 7)
+(5 rows)
+
+
+            WITH T1 as (SELECT * FROM cluster_test WHERE time=7),
+                 T2 as (SELECT * FROM expected WHERE time=7)
+            SELECT T1.time, T2.time, T1.temp, T2.temp, T1.location, T2.location,
+                   substring(T1.value for 30), substring(T2.value for 30),
+                   length(T1.value), length(T2.value)
+            FROM T1 FULL OUTER JOIN T2 ON T1 = T2
+            WHERE T1 IS NULL OR T2 IS NULL
+        
+ time | time | temp | temp | location | location | substring | substring | length | length 
+------+------+------+------+----------+----------+-----------+-----------+--------+--------
+(0 rows)
+
+COMMIT;
+-- Show chunk indexes
+SELECT * FROM test.show_indexes('_timescaledb_internal._hyper_1_1_chunk');
+                              Index                               |  Columns   | Expr | Unique | Primary | Exclusion | Tablespace 
+------------------------------------------------------------------+------------+------+--------+---------+-----------+------------
+ _timescaledb_internal._hyper_1_1_chunk_cluster_test_location_idx | {location} |      | f      | f       | f         | 
+ _timescaledb_internal._hyper_1_1_chunk_cluster_test_time_idx     | {time}     |      | f      | f       | f         | 
+(2 rows)
+
+SELECT * FROM test.show_indexes('_timescaledb_internal._hyper_1_2_chunk');
+                              Index                               |  Columns   | Expr | Unique | Primary | Exclusion | Tablespace 
+------------------------------------------------------------------+------------+------+--------+---------+-----------+------------
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_location_idx | {location} |      | f      | f       | f         | 
+ _timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx     | {time}     |      | f      | f       | f         | 
+(2 rows)
+
+\set ON_ERROR_STOP 0
+-- cannot move a chunk with no reorder index on first call 
+SELECT move_chunk(chunk=>'_timescaledb_internal._hyper_1_2_chunk', destination_tablespace=>'tablespace1', index_destination_tablespace=>'tablespace1');
+WARNING:  Timescale License expired
+ERROR:  there is no previously clustered index for table "_hyper_1_2_chunk"
+-- cannot move a chunk without a destination tablespace set
+SELECT move_chunk(chunk=>'_timescaledb_internal._hyper_1_2_chunk', index_destination_tablespace=>'tablespace1', reorder_index=>'_timescaledb_internal._hyper_1_2_chunk_cluster_test_time_idx');
+ERROR:  function move_chunk(chunk => unknown, index_destination_tablespace => unknown, reorder_index => unknown) does not exist at character 8
```

Comment:
this is not supposed to error like that is it?

Reply:
No, this is fine. It should fail in some way if it's not set, because it's a required argument, i think this is the way it will fail?

Reply:
eh, nvm, set it to null explicitly so it would trigger failure.